--- msn/CMakeLists.txt	2008-11-12 16:45:40.000000000 +0100
+++ msn/CMakeLists.txt-new	2008-11-26 23:07:39.033996250 +0100
@@ -42,6 +42,6 @@
 target_link_libraries(msn crypto)
 
 ########### install files ###############
-install(TARGETS msn DESTINATION  ${CMAKE_INSTALL_PREFIX}/lib)
+install(TARGETS msn DESTINATION  ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 install(FILES ${msn_HEADERS} DESTINATION ${CMAKE_INSTALL_PREFIX}/include/msn)
 
